STYLE_GUIDE.md

```diff
@@ -15,11 +15,25 @@ void formatted_code_again;
 ```
 
 #### Python
-Python code should conform to [PEP8](https://www.python.org/dev/peps/pep-0008/).
 
 Addons uses [flake8](http://flake8.pycqa.org/en/latest/) to check pep8 compliance and 
 code analysis.
 
+Addon use [Black](https://black.readthedocs.io/en/stable/) to format our code.
+The continuous integration check will fail if you do not use it.
+
+Install them with:
+```
+pip install flake8 black
+```
+
+Be sure to run them both before you push your commits, otherwise the CI will fail!
+
+```
+python -m black ./
+python -m flake8
```

Comment:
So at some point I would like to have this down to a single check. Once the rather large black reformatting is complete do you see a need for flake8? I believe as a subset of pep8 we should be fine (minus the line length)

Reply:
Flake8 is an addition of multiple tools. If we enforce black, then we should disable pycodestyle. See http://flake8.pycqa.org/en/latest/user/error-codes.html

I'd preferto use flake8 with pycodestyle disabled. We get useful checks and we can add plugins to flake8 in the future. 

I'll disable pycodestyle in another pull request if you don't mind :)